src/devices_models/devices/common/cost_functions.jl

```diff
@@ -270,15 +270,14 @@ function pwl_gencost_sos!(
     for i in 1:length(cost_data)
         pwlvars[i] = JuMP.@variable(
             container.JuMPmodel,
-            # TODO: Redefine the base_name for PWL vars.
-            # base_name = "{$(variable)}_{sos}",
+            base_name = "{$(variable)}_{sos_$(i)}",
```

Comment:
Did you test how does this look?

Reply:
this is how the variable name looks like , `ActivePowerVariable_ThermalStandard_{Alta, 2}_{pwl_3}` `ActivePowerVariable_ThermalStandard_{Alta, 2}_{sos_2}`